File: app/src/main/java/com/example/ign_app/articlemodel/data/Metadata.java
package com.example.ign_app.articlemodel.data;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Metadata {

    @SerializedName("headline")
    @Expose
    private String headline;

    @SerializedName("description")
    @Expose
    private String description;

    @SerializedName("slug")
    @Expose
    private String slug;


    @SerializedName("publishDate")
    @Expose
    private String publishDate;


    public String getHeadline() {
        return headline;
    }

    public void setHeadline(String headline) {
        this.headline = headline;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public String getPublishDate() {
        return publishDate;
    }

    public void setPublishDate(String publishDate) {
        this.publishDate = publishDate;
    }

    public String getSlug() {
        return slug;
    }

    public void setSlug(String slug) {
        this.slug = slug;
    }

    @Override
    public String toString() {
        return "Metadata{" +
                "headline='" + headline + '\'' +
                ", description='" + description + '\'' +
                ", slug='" + slug + '\'' +
                ", publishDate='" + publishDate + '\'' +
                '}';
    }
}

File: app/src/main/java/com/example/ign_app/articlemodel/data/Metadata.kt
package com.example.ign_app.articlemodel.data

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Metadata {
    @SerializedName("headline")
    @Expose
    var headline: String? = null

    @SerializedName("description")
    @Expose
    var description: String? = null

    @SerializedName("slug")
    @Expose
    var slug: String? = null

    @SerializedName("publishDate")
    @Expose
    var publishDate: String? = null

    override fun toString(): String {
        return "Metadata{" +
                "headline='" + headline + '\'' +
                ", description='" + description + '\'' +
                ", slug='" + slug + '\'' +
                ", publishDate='" + publishDate + '\'' +
                '}'
    }
}